txscript/standard_test.go

```diff
@@ -819,13 +819,6 @@ var scriptClassTests = []struct {
 			"5329a00357b3a7886211ab414d55a 1 CHECKMULTISIG",
 		class: MultiSigTy,
 	},
-	// tx e5779b9e78f9650debc2893fd9636d827b26b4ddfa6a8172fe8708c924f5c39d
-	{
```

Comment:
Why did we remove this?

Reply:
Seems like it was removed by mistake. Added it back.